src/main/java/com/mateacademy/table/Developer.java

```diff
@@ -0,0 +1,33 @@
+package com.mateacademy.table;
+
+import com.mateacademy.listener.MyListener;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(MyListener.class)
+@javax.persistence.Table(name = "developer")
+public class Developer extends TableX {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "age")
+    private Integer age;
+
+    @Column(name = "sex")
+    private String sex;
```

Comment:
We can represent "male" like "m", "man", ...
So let's try to be more specific. How can we do that?